src/Microsoft.ML.ImageAnalytics/ImageLoaderTransform.cs

```diff
@@ -200,7 +200,11 @@ protected override Delegate MakeGetter(IRow input, int iinfo, out Action dispose
                                 // REVIEW : Log failures.
                                 dst = null;
                             }
-                        }
+							var notFound = $"File {src.ToString()} was not found";
+							var invalidChar = $"File {src.ToString()} has an invalid character in its path and could not be loaded";
```

Comment:
>invalidChar [](start = 11, length = 11)

no need to create an error message every time, if you are not sure you're going to throw.
```
if (dst.PixelFormat...)
  throw Host.Except(...)
```

Reply:
Thanks, fixed!